src/Navbar/Navbar.stories.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { Dropdown } from 'react-bootstrap'
+import { storiesOf } from '@storybook/react'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import {
+  MenuItem,
+  Nav,
+  NavItem,
+  NavDropdown,
+  Navbar,
+  NavbarBrand,
+  NavbarCollapse,
+  NavbarHeader,
+  NavbarToggle
+} from '../index.js'
+
+const stories = storiesOf('Navbar', module)
+
+const description = (
+  <p>
+    This component is based on React Bootstrap Navbar component. See{' '}
+    <a href="https://react-bootstrap.github.io/components.html#navbars">
+      React Bootstrap Docs
+    </a>{' '}
+    for complete Navbar component documentation.
+  </p>
+)
+
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Navbar',
+    description: description,
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/navigation/horizontal-navigation/'
+  })
+)
+
+stories.addWithInfo('Navbar', '', () =>
+  <div>
+    <Navbar className="navbar navbar-default navbar-pf">
+      <NavbarHeader>
+        <NavbarBrand>
+          <img src="img/brand.svg" alt="PatternFly Enterprise Application" />
+        </NavbarBrand>
+        <NavbarToggle />
+      </NavbarHeader>
+      <NavbarCollapse>
+        <Nav bsClass="nav navbar-nav navbar-utility">
+          <Dropdown id="helpDropdown" componentClass="li" bsClass="dropdown">
```

Comment:
What is the difference between this Dropdown and NavDropdown used below?

Reply:
Unfortunately, there were style differences when the `NavDropdown` was used (React Bootstrap's component was not all that flexible). I'm leaning towards just using `Dropdown` only (it seems more versatile), or writing our own `NavDropdown` component. WDYT?